crates/ruff_linter/src/rules/numpy/snapshots/ruff_linter__rules__numpy__tests__numpy2-deprecation_NPY201.py.snap

```diff
@@ -0,0 +1,865 @@
+---
+source: crates/ruff_linter/src/rules/numpy/mod.rs
+---
+NPY201.py:4:5: NPY201 [*] `np.add_docstring` will be removed in NumPy 2.0. Use `numpy.lib.add_docstring` instead.
+  |
+2 |     import numpy as np
+3 | 
+4 |     np.add_docstring
+  |     ^^^^^^^^^^^^^^^^ NPY201
+5 | 
+6 |     np.add_newdoc
+  |
+  = help: Use `numpy.lib.add_docstring` instead.
+
+ℹ Fix
+  1 |+from numpy.lib import add_docstring
+1 2 | def func():
+2 3 |     import numpy as np
```

Comment:
@charliermarsh, here is an example.

Reply:
I think that instead we should focus on trying to get this to use `np.lib.add_docstring` here.

Reply:
I think this is a great way and will only change the line where changes are needed.  

Just needs to be sure that the different submodules are initialized when running `import numpy as np`.